docs/src/install-github-ci.md

```diff
@@ -6,7 +6,7 @@ CI.
 
 ## Using Kani in your GitHub workflow
 Our GitHub Action is available in the GitHub Marketspace with the name
```

Comment:
Marketspace -> Marketplace

Also, what if instead of saying the name you link to the Marketplace?